### PR TITLE
Add Travis jobs for LDAP

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@anusudarsan @amrutagokhale

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +45,14 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
         singlenode-kerberos-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation,authorization
   - |
+    test ! -v PRODUCT_TESTS ||
+      presto-product-tests/bin/run_on_docker.sh \
+        singlenode-ldap -g ldap_cli
```
Comment (amrutagokhale) [inline .travis.yml]: what about `ldap` group? Or is it that `ldap` is a subset of `ldap_cli`?

**Comment by anusudarsan** [comment]
lgtm

**Comment by amrutagokhale** [comment]
Looks good.

**Comment by petroav** [comment]
There are 5 commits in this pull request that have little to do with Travis. Did you mean to only include your last commit?

**Comment by anusudarsan** [comment]
I just rebased the branch again, to just show one relevant commit

File: presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'teradatalabs/centos6-java8-oracle-ldap:5'
+    command: /docker/volumes/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    extra_hosts:
+       - "${LDAP_SERVER_HOST}:${LDAP_SERVER_IP}"
```
Comment (ArturGajowy) [inline presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml:8]: I don't see anything setting those two variables. Shouldn't they have some default values when running on docker?

**Comment by sanjay990** [comment]
Already merged

**Pull request closed without merging**